### PR TITLE
Sql query deployment

### PR DESCRIPTION
**Pull request by TheSlimvReal**
- Adds the [query backend](https://github.com/Aam-Digital/query-backend) and a container for the [SQS](https://neighbourhood.ie/products-and-services/structured-query-server) to the app deployment
- Added instructions on how these services can be activated

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.18.4
+COPY ./structured-query-server sturctured-query-server
```
Comment (sleidig) [inline Dockerfile:2]:
```suggestion
COPY ./structured-query-server structured-query-server
```
?

File: README.md [hunk 2]
```diff
@@ -108,6 +108,17 @@ There are many ways in which the authentication flow can be configured.
...
+# Reporting with SQS
+It is possible to create reports on the apps data using SQL queries.
+For this we have integrated the [structured-query-service (SQS)](https://neighbourhood.ie/products-and-services/structured-query-server).
+This creates a read-only copy of the data in the CouchDB and allows to run Sqlite queries against it.
+
+To activate it for an application, simply add `SQS_PROFILE=backend` to the applications `.env` file and run `docker compose up -d`.
+
+The SQS executable needs to be placed in the setup folder (next to the `Dockerfile`).
+The path to the `Dockerfile` can be adjusted by adding `SETUP_DIR=/path/to/setup/dir` to the applications `.env` file. 
+Default is `/var/docker/setup`.
```
Comment (sleidig) [inline README.md:120]: Do you not have to build that Dockerfile also?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.18.4
```
Comment (sleidig) [inline Dockerfile:1]: can we add a comment here on top, explaining the role of this docker file?

**Review by sleidig — COMMENTED** [review]
I didn't test this (as we don't have SQS yet, also) but the code looks clear enough

File: README.md (hunk 2 above)
Comment (TheSlimvReal) [reply README.md:120]: No, it builds automatically if missing. It just needs to know where to look for the Dockerfile.

**Pull request closed without merging**